scripts/ports.cmake

```
@@ -149,6 +149,9 @@ if(CMD STREQUAL "BUILD")
         if(VCPKG_FIXUP_ELF_RPATH)
             include("${SCRIPTS}/cmake/z_vcpkg_fixup_rpath.cmake")
         endif()
+        if(VCPKG_FIXUP_MACHO_RPATH)
+            include("${SCRIPTS}/cmake/z_vcpkg_fixup_install_name_rpath.cmake")
+        endif()
```

```suggestion
        z_vcpkg_fixup_install_name_rpath_in_dir()
```

and include unconditionally at the top. The function should always be available.

---

Does that imply that linux rpath fixup mechanism should also be included unconditionally?

---

So let `z_vcpkg_fixup_install_name_rpath_in_dir()` check the running triplet like when hitting `*-osx-dynamic` to run the fixup?